sdk/build.gradle

```diff
@@ -8,6 +8,7 @@ plugins {
 
 description = 'OpenTelemetry SDK'
 ext.moduleName = "io.opentelemetry.sdk"
+ext.propertiesDir = "src/main/resources/io/opentelemetry/sdk"
```

Comment:
Don't generate to source, generate to build, e.g. `build/generated/properties/io/opentelemetry/sdk`